spec/models/spree/order_spec.rb

```diff
@@ -195,6 +195,21 @@
       expect(order.updater).to receive(:before_save_hook)
       order.finalize!
     end
+
+    context "extra payments exist that require authorization" do
+      let!(:cash_payment) { build(:payment, state: "completed", amount: order.new_outstanding_balance) }
+      let!(:stripe_payment) { build(:payment, state: "requires_authorization") }
+      before do
+        order.payments << cash_payment
+        order.payments << stripe_payment
+        allow_any_instance_of(Spree::Payment).to receive(:void_transaction!) {}
```

Comment:
This line can be removed. The spec passes anyway. It's actually better removed because then the spec fails if both payments receive `void_transaction`.
```suggestion
```